backend/src/main/kotlin/org/loculus/backend/service/submission/EmptyProcessedDataProvider.kt

```diff
@@ -11,15 +11,16 @@ import org.springframework.stereotype.Component
 class EmptyProcessedDataProvider(private val backendConfig: BackendConfig) {
     fun provide(organism: Organism): ProcessedData<GeneticSequence> {
         val (schema, referenceGenomes) = backendConfig.getInstanceConfig(organism)
+        val referenceGenome = referenceGenomes.values.first()
 
-        val nucleotideSequences = referenceGenomes.nucleotideSequences.map { it.name }.associateWith { null }
+        val nucleotideSequences = referenceGenome.nucleotideSequences.map { it.name }.associateWith { null }
```

Comment:
I think this is confusing - rather than renaming what we call "referenceGenomes" we should iterate over "subOrganisms" higher up